### PR TITLE
Update the 4.2 EO and CLO installs

### PR DESCRIPTION
**Pull request by mburke5678**
Per Qiaoling email Sep 5, 2019, 9:01 PM

### CODE REVIEW

**Comment by mburke5678** [comment]
@QiaolingTang Please take a look.

File: modules/efk-logging-deploy-subscription.adoc [hunk 1]
```diff
@@ -181,11 +182,11 @@ metadata:
 spec:
   channel: "preview" <2>
```
Comment (QiaolingTang) [inline modules/efk-logging-deploy-subscription.adoc]: The channel name is `"4.2"`, https://github.com/openshift/elasticsearch-operator/blob/release-4.2/manifests/elasticsearch-operator.package.yaml#L4

**Comment by QiaolingTang** [comment]
LGTM, thanks.

**Comment by mburke5678** [comment]
@QiaolingTang Thank you for the review!

**Pull request merged**